guides/source/testing.md

```diff
@@ -758,13 +758,17 @@ default. Loading involves three steps:
 2. Load the fixture data into the table
 3. Dump the fixture data into a method in case you want to access it directly
 
-TIP: In order to remove existing data from the database, Rails tries to disable
-referential integrity triggers (like foreign keys and check constraints). If you
-are getting permission errors on running tests, make sure the database user has
-the permission to disable these triggers in the testing environment. (In
-PostgreSQL, only superusers can disable all triggers. Read more about
-[permissions in the PostgreSQL
-docs](https://www.postgresql.org/docs/current/sql-altertable.html)).
+TIP: In order to delete and insert fixtures' data into the tables, Rails
+disables referential integrity triggers (like foreign keys and check
```

Comment:
I don't think this change is necessary

Reply:
thx for the feedback @zzak! I was trying to be more explicit clarifying that Rails disables triggers not just for deleting existing data, but also for inserting it

Reply:
This is just my feeling, but trying to cover all of the corner cases in this section is a bit much. We should just focus on introducing the core concept of fixtures in the context of Rails test strategy here.

Reply:
I totally see your point, appreciate it! Do you think it'd be helpful to add some clarification about this fixtures/referential-integrity topic somewhere in the API docs instead? Potentially in [activerecord/lib/active_record/fixtures.rb](
https://github.com/rails/rails/blob/17cde11cd13cf07a7d6c712cb1593881496abacb/activerecord/lib/active_record/fixtures.rb#L13-L526).